setup.py

```diff
@@ -67,6 +67,7 @@
     you need to go back to main before executing this.
 """
 
+
```

Comment:
This extra line triggers a cache miss in GHA CI, which fetches the updated hub version.